advisories/0011.md

```diff
@@ -0,0 +1,57 @@
+### Mirantis Kubernetes Engine (MKE) and CVE-2024-21626: Understanding the Impact and Mitigation
+#### Release Date
+2024-02-08
+#### Overview
+This advisory addresses the impact of CVE-2024-21626, a critical vulnerability identified within runc, and its implications for the Mirantis Kubernetes Engine (MKE). Given that MKE relies heavily on the Mirantis Container Runtime (MCR), this vulnerability directly impacts its operational integrity. We aim to clarify the situation, explain the mitigation steps, and provide actionable guidance for MKE users.
+
+### Vulnerability Information
+#### CVE Identifier
+CVE-2024-21626
+#### CVSSv3.1
+8.6 (High) CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:C/C:H/I:H/A:H
+#### CWEs
+CWE-403, CWE-668
+
+#### Affected Product
+* MKE deployment requires installation of MCR.  Older versions of MCR (<= 23.0.9) contain a vulnerability in the underlying runc component. Please see [the MCR advisory](https://github.com/Mirantis/security/blob/main/advisories/0010.md) for details. Any deployment of MKE with MCR version <= 23.0.9 must upgrade to MCR 23.0.9-1 or later to mitigate this risk.
```

Comment:
```suggestion
* MKE deployment requires installation of MCR.  Older versions of the MCR packages contain a vulnerable version of the runc component. Please see [the MCR advisory](https://github.com/Mirantis/security/blob/main/advisories/0010.md) for details.
```

The vulnerable versions and patching steps have some nuance that is neither correctly nor adequately captured here.

Reply:
While you could technically install individual packages like Docker-EE, Docker-EE-CLI, and containerd, Mirantis does not officially support such a configuration. Customers and Mirantis typically and should always treat MCR as a single, integrated product that includes all these components pre-configured and tested for compatibility. Using MCR ensures a supported and well-maintained environment, avoiding potential issues arising from manual package installation and management.